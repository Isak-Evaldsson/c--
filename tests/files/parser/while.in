int func1() {
    // Empty
    while (1) {}

    // Block
    while (1) {
        int a;
        print();
    }

    // No-Block
    while(1)
        return 1;
}