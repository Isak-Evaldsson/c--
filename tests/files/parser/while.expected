Prototype: func1
    While:
        LiteralExpr: 1
    Do:
    While:
        LiteralExpr: 1
    Do:
        VarDecl: a
        FunCall: print
    While:
        LiteralExpr: 1
    Do:
        Return 
            LiteralExpr: 1
